### ISSUE TITLE
Enable caching

### ISSUE THREAD

**Issue by twodayslate** [twodayslate/gateway#11]
Add a new header that allows for the option to cache data and use the cached response instead of the actual one

**Comment by twodayslate** [comment]
Oh apparently you can also do this with cache rules

**Comment by KamaniBhavin** [comment]
@twodayslate, Is development required for this? I can incorporate a caching layer to cache data for a specified duration.

**Comment by twodayslate** [comment]
Yeah match and put could be used. Should probably only put 200s and if it has an X-gateway header
